tensorflow/core/kernels/rnn/BUILD

```diff
@@ -19,7 +23,11 @@ licenses(["notice"])  # Apache 2.0
 
 tf_gpu_library(
     name = "blas_gemm",
-    srcs = if_cuda_is_configured(["blas_gemm.cc"]),
+    srcs = [] + if_cuda_is_configured([
```

Comment:
`[] +` looks unnecessary.

Reply:
Agree with that. Though if I don't have it, `buildifier` will instead format it as the following:
```
    src = if_cuda_is_configured(["blas_gemm.cc"]) + if_rocm_is_configured([
          "blas_gemm.cc"
    ]),
```
Which is more ugly than how it looks now.